src/defs.ts

```diff
@@ -91,16 +93,13 @@ export interface ChannelBasicInfo {
   id?: string;
   name?: string;
   status?: Status;
-  description?: string;
 }
 
 export interface Channel extends ChannelBasicInfo {
   domain_id?: string | DomainBasicInfo;
   metadata?: Record<string, any>;
   tags?: string[];
-  ParentGroup?: string;
-  level?: number;
-  path?: string;
+  parent_group_id?: string;
   created_at?: Date;
   updated_at?: Date;
   updated_by?: string;
```

Comment:
It also has permissions:
https://github.com/absmach/magistrala/blob/9bf293fd44bbeceb7ac6409c7cd12bb9caac8f74/channels/channels.go#L29

Reply:
https://github.com/absmach/sdk-js/blob/c39b9c2590b24adb3484c53216eca0301c1b4c5c/src/defs.ts#L106